src/components/README.md

```diff
@@ -95,6 +95,8 @@ ReactDOM.render(app, domContainerNode);
 
 ### CSS components
 
+Add the [latest CSS variables](https://unpkg.com/browse/@shopify/polaris-tokens@latest/dist-modern/palette/base.light.polaris.custom-properties.css) from the polaris tokens library.
```

Comment:
What does this mean from an implementation perspective??

I was going to go for this approach in my project, but perhaps there's another recommended way?

```html
    <link
      rel="stylesheet"
      href="https://unpkg.com/@shopify/polaris-tokens@2.18.0/dist-modern/palette/base.dark.polaris.custom-properties.css"
    />
    <link
      rel="stylesheet"
      href="https://unpkg.com/@shopify/polaris@6.0.0-alpha.7/dist/styles.css"
    />
```